packages/qwik/src/optimizer/src/plugins/vite.ts

```diff
@@ -105,6 +105,22 @@ export function qwikVite(qwikViteOpts: QwikVitePluginOptions = {}): any {
 
       qwikPlugin.log(`vite config(), command: ${viteCommand}, env.mode: ${viteEnv.mode}`);
 
+      if (sys.env === 'node' && !qwikViteOpts.entryStrategy) {
```

Comment:
I used `fs.promise` and I saw in other part of the codebase that is limited to sys.env === 'node'
How can we cover other environments (eg. deno, bun) ?

Reply:
No idea, I don't think we need to solve that now.